README.md

```diff
@@ -12,6 +12,25 @@ APIs with a consistent API look.
 qLibc is published under 2-clause BSD license known as Simplified BSD License.
 Please refer the LICENSE document included in the package for more details.
 
+## Installation
+
+qlibc uses GNU AutoTools as the primary tool for compilation. Simply execute the following commands while in the main directory of the project.
+
+```bash
+./configure
+make
+sudo make install
+```
+
+Alternatively, you can use Cmake. qLibc requires at least GNU99 and confirmed to be compiling in GNU23
```

Comment:
I expect all the Linux/Linux-like users to use autotools and CMake for Windows.
Do you think we should include Windows installation instructions instead?